webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js

```diff
@@ -217,3 +217,29 @@ describe('Without tracer installed', () => {
     expect(enableTracesModalLink).toHaveClass('pf-m-in-progress');
   });
 });
+
+test('verify banner is present for old host page', async (done) => {
```

Comment:
This should be moved to the 'With Tracer installed' section

Reply:
hey got the feedback addressed on the pr except for this one, can you explain a bit more where to move it to? https://github.com/Katello/katello/pull/9691#discussion_r721595905 I see you have this setup but I didnt think you could put a test inside of here:
https://github.com/Katello/katello/blob/master/webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js#L48